src/amr/tagging/default_hybrid_tagger_strategy.hpp

```diff
@@ -17,8 +16,17 @@ class DefaultHybridTaggerStrategy : public HybridTaggerStrategy<HybridModel>
     using gridlayout_type           = typename HybridModel::gridlayout_type;
     static auto constexpr dimension = HybridModel::dimension;
 
+
 public:
+    DefaultHybridTaggerStrategy(initializer::PHAREDict const& dict)
+        :threshold_{[&](){return (dict.contains("threshold")) ? dict["threshold"].template to<double>() : 0.1;}()}
```

Comment:
you don't want to use `cppdict::get_value(dict, "threshold", 0.1),`  ?

also the lambda is redundant `dict.contains("threshold") ? dict["threshold"].template to<double>() : 0.1;` is the same